docs/setup.md

```diff
@@ -1,18 +1,23 @@
 
 # Setting up ClintLR
 
-ClintLR is a desktop Java application that includes LIRICAL analysis. Therefore, ClintLR requires LIRICAL to be installed.
-Instructions for installing LIRICAL can be found [here](https://thejacksonlaboratory.github.io/LIRICAL/stable/setup.html#rstsetup).
-Similar to LIRICAL, ClintLR also requires Exomiser to be installed as a library before it can be compiled and built.
-
-
 ## Prerequisites
 
 ClintLR was written with Java version 17 but will compile under Java 17 or higher. If you want to
 build ClintLR from source, then the build process described below requires
 [Git](https://git-scm.com/book/en/v2) and
 [Java Development Kit 17](https://www.oracle.com/java/technologies/downloads/) or higher.
+ClintLR also requires Exomiser to be installed as a library before it can be compiled and built.
+
 
+## Installing ClintLR
```

Comment:
I think you should split the installation into two subsections:
- Download the prebuilt JAR file
- Build from sources

We must make it clear that the first section should be done most of the time. We also need to point out that at the end of any of the sections you will get the same result - a runnable GUI JAR file.

Reply:
Ok, thanks.